docs/ConfigureScalarDL.md

```diff
@@ -0,0 +1,24 @@
+# Configure Scalar DL 
+
+This guide explains how to configure helm charts for Scalar DL deployment.
+
+## Steps
```

Comment:
This doc is for configuring Scalar DL overall or only for database-related configurations?
The current doc looks like the latter but it doesn't say so, so users would confuse.
Please explain it properly and claerly.

Reply:
Updated description in 4791dc7